CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
  
 #### Fixes
 
+* [#1438](https://github.com/ruby-grape/grape/pull/1439): Fix for default param modification bug - [@jlfaber](https://github.com/jlfaber).
```

Comment:
Lets write something more useful, maybe "Default values in `params` are now cloned on assignment"?